forge-gui/res/cardsfolder/upcoming/roving_harper.txt

```diff
@@ -0,0 +1,8 @@
+Name:Roving Harper
+ManaCost:2 W
+Types:Creature Elf Scout
+PT:2/2
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDraw | TriggerDescription$ When CARDNAME enters the battlefield, draw a card.
+SVar:TrigDraw:DB$ Draw | Defined$ You | NumCards$ 1
```

Comment:
`Defined$ You | NumCards$ 1` are defaults, so they aren't strictly needed.

Just FYI – if you prefer to have them there, you're welcome to just resolve this conversation 😉